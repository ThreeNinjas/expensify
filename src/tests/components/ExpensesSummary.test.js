import React from 'react'
import {shallow} from 'enzyme'
import expenses from '../fixtures/expenses'
import {ExpensesSummary} from '../../components/ExpensesSummary'

test('should correctly render ExpenesSummary with one expense', () => {
     const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235}/>)
     expect(wrapper).toMatchSnapshot()
})

test('should correctly render ExpenesSummary with multiple expenses', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={235234313234313}/>)
    expect(wrapper).toMatchSnapshot()
})