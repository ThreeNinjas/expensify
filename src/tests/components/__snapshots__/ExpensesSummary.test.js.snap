// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ExpenesSummary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totaling 
    $2,352,343,132,343.13
  </h1>
</div>
`;

exports[`should correctly render ExpenesSummary with one expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totaling 
    $2.35
  </h1>
</div>
`;
